constants.py

```diff
@@ -20,3 +20,19 @@
     "{run}.0{version}/{date:%Y}/{date:%m}/3B-DAY-{run}.MS.MRG.3IMERG."
     "{date:%Y%m%d}-S000000-E235959.V0{version}{version_letter}.nc4"
 )
+
+OUTPUT_METADATA = {
```

Comment:
Also, if these are to be the standard metadata for all rasters, it would be good to include a `version`. For ERA5 and SEAS5 I believe this is irrelevant, so can just leave as `None`. But for IMERG it's relevant, and for some datasets (e.g. GloFAS), the version is super important.

However if you'd prefer we can add this later since it's not relevant for ERA5 and SEAS5.

Reply:
Yes great points! I like both of these